distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -159,7 +159,7 @@ type Array
          If a `Range`, the selection is specified by two indices, from and to.
     @range Index_Sub_Range.default_widget
     take : (Index_Sub_Range | Range | Integer) -> Vector Any
-    take self range=(Index_Sub_Range.First 1) = Vector.take self range
+    take self range=(Index_Sub_Range.First 1) = (Vector.from_polyglot_array self).take range
```

Comment:
What's the effect of `Vector.from_polyglot_array` wrapping on engine benchmarks? I am afraid some of they might slow down.

Wrapping all `Array` objects by `from_polyglot_array` isn't the expected fix. I was expecting all the static methods to move into `Array_Like_Helpers` and **both** the `Array` and `Vector` instance methods to delegate to the `Array_Like_Helpers` implementations.

Reply:
I did implement the `Array_Like_Helpers` solution in the first place, but @jdunkerley suggested the wrapping one might be better.

If we decide to go back to `Array_Like_Helpers`, it's just the matter of reverting the commit [a8171168789d7252478cc334d82b2e29e51ee024](https://github.com/enso-org/enso/commit/a8171168789d7252478cc334d82b2e29e51ee024)

I guess either way we should wait for the fix of engine benchmarks and see how both solutions compare there.

I'm happy to go for either of these.